README.md

```diff
@@ -2,16 +2,24 @@
     <a href="https://sentry.io" target="_blank" align="center">
         <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
     </a>
-<br/>
-    <h1>Gradle plugin for Sentry Android</h1>
 </p>
 
+_Bad software is everywhere, and we're tired of it. Sentry is on a mission to help developers write better software faster, so we can get back to enjoying technology. If you want to join us [<kbd>**Check out our open positions**</kbd>](https://sentry.io/careers/?utm_source=github&utm_medium=readme&utm_campaign=sentry-android-gradle-plugin)_
+
+Gradle Plugin for Sentry Android
+===========
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Test%20Publishing%20UUID%20Dry%20Run/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Test%20Publishing%20UUID/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Pre%20Merge%20Checks/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![line count](https://tokei.rs/b1/github/getsentry/sentry-android-gradle-plugin/)](https://github.com/getsentry/sentry-android-gradle-plugin/tree/main)
+[![Discord Chat](https://img.shields.io/discord/621778831602221064?logo=discord&logoColor=ffffff&color=7389D8)](https://discord.gg/PXa5Apfe7K)  
```

Comment:
this is already under `Resources`, should we remove one of them?

Reply:
Because there's not much on this readme it's obvious, in other repos it's there but one doesn't notice: 
![image](https://user-images.githubusercontent.com/1633368/118384954-93132180-b5d8-11eb-8f1b-6b8478c134ac.png)

![image](https://user-images.githubusercontent.com/1633368/118384957-973f3f00-b5d8-11eb-966a-247f629e46a2.png)

I don't mind removing from either side in here.